com.unity.shadergraph/Editor/Data/Graphs/SerializableCubemap.cs

```diff
@@ -19,21 +25,26 @@ public Cubemap cubemap
         {
             get
             {
-                if (string.IsNullOrEmpty(m_SerializedCubemap))
-                    return null;
+                if (!string.IsNullOrEmpty(m_SerializedCubemap))
+                {
+                    var textureHelper = new CubemapHelper();
+                    EditorJsonUtility.FromJsonOverwrite(m_SerializedCubemap, textureHelper);
+                    m_SerializedCubemap = null;
+                    m_Guid = AssetDatabase.AssetPathToGUID(AssetDatabase.GetAssetPath(textureHelper.cubemap));
+                    m_Cubemap = textureHelper.cubemap;
+                }
+                else if (!string.IsNullOrEmpty(m_Guid) && m_Cubemap == null)
+                {
```

Comment:
Could probably remove the curly brackets

Reply:
Intentional, as the other block in the if above uses curly braces.